# Configuration pour la communication avec le servo controller
servo_controller:
  port: "/dev/ttyACM1"  # Port série pour le servo controller (différent du joystick)
  baudrate: 115200
  timeout: 1

servo_x:
  neutral: 85
  min: 70
  max: 100
servo_y:
  neutral: 92
  min: 75
  max: 105
amplitude: 15

emergency_timeout: 0.5  # Temps avant retour à position neutre en cas d'urgence
